src/gentropy/method/pics.py

```diff
@@ -160,6 +160,10 @@ def _finemap(
                 # If PICS cannot be calculated, we drop the variant from the credible set
                 continue
 
+            # Chaing chema:
```

Comment:
Is this accidental?

Reply:
This PR also fixes the variantId null problem in the PICSed locus.